projects/cashmere-examples/src/lib/datepicker/datepicker-example.component.ts

```diff
@@ -1,15 +1,10 @@
-import {Component, OnInit} from '@angular/core';
+import {Component} from '@angular/core';
 
 @Component({
     selector: 'hc-datepicker-example',
     templateUrl: './datepicker-example.component.html',
     styleUrls: ['datepicker-example.component.scss']
 })
-export class DatepickerExampleComponent implements OnInit {
-    constructor() {}
-
+export class DatepickerExampleComponent {
     date1 = new Date(2010, 1, 1);
-    date2 = new Date(2010, 1, 1);
-
-    ngOnInit() {}
 }
```

Comment:
Simplifying up the example ts - there was old code in here that wasn't being used.